howso/engine/client.py

```diff
@@ -9,7 +9,7 @@
 }
 
 
-def get_client() -> AbstractHowsoClient:
+def get_client():
```

Comment:
Type hints were removed from the methods of this class. Should add them back.

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60

Reply:
Still missing type hints on `use_client`. Also, can you fix the `__all__` module var while you are in here. Should be a list not a set.